### ISSUE TITLE
Suttle differences between replica and Original for Clearleft Podcast on Firefox

### ISSUE THREAD

**Issue by FrenzyExists** [Software-clones/frontend#2]
TL;DR Firefox doesn't seem to display well the size of some things, as shown in the pics. Still its a pretty darn good replica, if the differences in the picture are fixed, it would be a perfect replica of it, pretty darn impressive!

## Original Site
![image](https://user-images.githubusercontent.com/47986470/160034051-311c6038-3937-4291-aa72-46fc23bec32b.png)

![image](https://user-images.githubusercontent.com/47986470/160034091-c8f8afa0-dde0-4983-853b-dd459a1a7141.png)

## Replica
![image](https://user-images.githubusercontent.com/47986470/160034423-450e8a94-8dcb-49da-bacc-65f826a1919a.png)

![image](https://user-images.githubusercontent.com/47986470/160034157-7bcafc5a-39bb-432c-a352-d1df352be711.png)

### Issues:
- Firefox doesn't apply the same font
- Some size in icons and font for for some reason, on firefox
- The original site's scrollbar is a diferent color, shown here

### Replica:
![image](https://user-images.githubusercontent.com/47986470/160034581-70f682b8-3824-4892-a655-eddff8877c01.png)

### Original
![image](https://user-images.githubusercontent.com/47986470/160034622-e5cdb6b5-fa22-4884-817f-4abff4afe4da.png)

**Comment by cloneables** [comment]
@FrenzyExists  Good catch!
I will assign someone on the team to fix this aSAP.
Feel free to submit a PR if you have the bandwidth!🙂
